app/models/goods_nomenclature.rb

```diff
@@ -69,4 +69,15 @@ def rules_of_origin_rules(...)
   def self.is_heading_id?(goods_nomenclature_item_id)
     goods_nomenclature_item_id.ends_with?('000000') && goods_nomenclature_item_id.slice(2, 2) != '00'
   end
+
+  def inspect
+    {
+      object_id: object_id,
+      id:,
+      type:,
+      goods_nomenclature_item_id:,
+      formatted_description:,
```

Comment:
we don't need this, I'll remove it!

Reply:
My view is we should override ApiEntity an use accumulated attr_accessors